pep-0621.rst

```diff
@@ -418,7 +416,7 @@ thus becomes a corresponding ``Requires-Dist`` entry for the matching
 Specifies which fields listed by this PEP were intentionally
 unspecified so another tool can/will provide such metadata
 dynamically. This clearly delineates which metadata is purposefully
-unspecified and expected to stay unspecified compared to being
+unspecified and expected to stay unspecified compared versus being
```

Comment:
"compared to" or "versus" is fine, but not both 😉 I actually prefer the original wording.